<!DOCTYPE html> <html lang="fr"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Frédéric Grandjean </title> <meta name="author" content="Frédéric Grandjean"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="grandjean, EBI, ADN environnemental, relations hôtes-parasite, génétique de la conservation, bioindicateurs"> <link rel="stylesheet" href="/TEST/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/TEST/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/TEST/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/TEST/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/TEST/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ltho-admin.github.io/TEST/"> <script src="/TEST/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/TEST/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/TEST/">A propos <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/TEST/publications/">Publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/TEST/projects/">Recherche </a> </li> <li class="nav-item "> <a class="nav-link" href="/TEST/cv/">CV </a> </li> <li class="nav-item "> <a class="nav-link" href="/TEST/teaching/">Enseignements </a> </li> <li class="nav-item "> <a class="nav-link" href="/TEST/people/">Collaborations </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Frédéric</span> Grandjean </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/TEST/assets/img/prof_pic-480.webp 480w,/TEST/assets/img/prof_pic-800.webp 800w,/TEST/assets/img/prof_pic-1400.webp 1400w," type="image/webp" sizes="(min-width: 930px) 270.0px, (min-width: 576px) 30vw, 95vw"> <img src="/TEST/assets/img/prof_pic.jpg?7492a90e9cc75cd0a7759a62ac1b8b05" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>Bâtiment B31</p> <p>3 Rue Jacques Fort - TSA 51106</p> <p>86073 POITIERS Cedex 9 </p> </div> </div> <div class="clearfix"> <p>Je suis Professeur des Universités à l’Université de Poitiers, au sein de l’équipe Écologie, Évolution, Symbiose (EES) du laboratoire <a href="https://ebi.labo.univ-poitiers.fr" rel="external nofollow noopener" target="_blank">Écologie et Biologie des Interactions (EBI – UMR 7267)</a>. Mes travaux s’inscrivent dans le domaine de la biologie de la conservation et de la génétique des populations, en mobilisant des approches intégrées allant de la génétique des populations à l’ADN environnemental.</p> <p>Mes recherches visent à décrire et comprendre la structuration génétique des populations animales et à évaluer l’impact des activités anthropiques et des processus naturels sur la biodiversité, notamment chez les écrevisses, iguanes et tritons. Je m’intéresse également aux interactions entre espèces invasives et espèces natives, en combinant des approches expérimentales, omiques et écologiques.</p> <p>Je coordonne ou participe à plusieurs projets nationaux et internationaux sur la conservation des écrevisses autochtones, la surveillance environnementale par métabarcoding et l’évolution génomique des symbioses.</p> </div> <h2> <a href="/TEST/news/" style="color: inherit">News</a> </h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Oct 15, 2025</th> <td> Premier test de mise en ligne du site ! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"><img class="emoji" title=":nerd_face:" alt=":nerd_face:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f913.png" height="20" width="20"> </td> </tr> </table> </div> </div> <h2> <a href="/TEST/publications/" style="color: inherit">Publications représentatives</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gaulin_genomics_2018" class="col-sm-8"> <div class="title">Genomics analysis of Aphanomyces spp. identifies a new class of oomycete effector associated with host adaptation</div> <div class="author"> Elodie Gaulin, Michiel J. C. Pel, Laurent Camborde, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Hélène San-Clemente, Sarah Courbier, Marie-Alexane Dupouy, Juliette Lengellé, Marine Veyssiere, Aurélie Le Ru, Frédéric Grandjean, Richard Cordaux, Bouziane Moumen, Clément Gilbert, Liliana M. Cano, Jean-Marc Aury, Julie Guy, Patrick Wincker, Olivier Bouchez, Christophe Klopp, Bernard Dumas' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>BMC Biology</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1186/s12915-018-0508-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="38175197"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12915-018-0508-5" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Oomycetes are a group of filamentous eukaryotic microorganisms that have colonized all terrestrial and oceanic ecosystems, and they include prominent plant pathogens. The Aphanomyces genus is unique in its ability to infect both plant and animal species, and as such exemplifies oomycete versatility in adapting to different hosts and environments. Dissecting the underpinnings of oomycete diversity provides insights into their specificity and pathogenic mechanisms.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">gaulin_genomics_2018</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Genomics analysis of {Aphanomyces} spp. identifies a new class of oomycete effector associated with host adaptation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{16}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1741-7007}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s12915-018-0508-5}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s12915-018-0508-5}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2025-10-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{BMC Biology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gaulin, Elodie and Pel, Michiel J. C. and Camborde, Laurent and San-Clemente, Hélène and Courbier, Sarah and Dupouy, Marie-Alexane and Lengellé, Juliette and Veyssiere, Marine and Le Ru, Aurélie and Grandjean, Frédéric and Cordaux, Richard and Moumen, Bouziane and Gilbert, Clément and Cano, Liliana M. and Aury, Jean-Marc and Guy, Julie and Wincker, Patrick and Bouchez, Olivier and Klopp, Christophe and Dumas, Bernard}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Aphanomyces, Effector, Host adaptation, Oomycete, Secretome, SSP}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="tan_giant_2020" class="col-sm-8"> <div class="title">A Giant Genome for a Giant Crayfish (Cherax quadricarinatus) With Insights Into cox1 Pseudogenes in Decapod Genomes</div> <div class="author"> Mun Hua Tan, Han Ming Gan, Yin Peng Lee, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Frederic Grandjean, Laurence J. Croft, Christopher M. Austin' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Frontiers in Genetics</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fgene.2020.00201" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="76523082"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2020.00201" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>1. IntroductionThe Decapoda represent a highly speciose and diverse group of crustaceans that provide important ecosystem services across mainly marine and freshwater environments, with many of the larger species being of significant commercial importance for fisheries and aquaculture industries. The value of generating and using genomic resources for key crustacean species and groups is widely appreciated for a variety of purposes including phylogenetic and population studies, selective breeding programs and broodstock management (Tan et al., 2016; Grandjean et al., 2017; Wolfe et al., 2019; Zenger et al., 2019; Zhang et al., 2019). However, comprehensive genomic studies are particularly limited for decapod crustaceans (Zenger et al., 2019; Zhang et al., 2019), compared to other aquatic invertebrate groups. Most genomic-related studies on decapod crustaceans have focused on mitogenome recovery and transcriptomics studies, with few genome assemblies attempted and most of these on commercially-important species (Song et al., 2016; Yuan et al., 2018; Zhang et al., 2019). Of the decapod species for which genome assemblies have been published and are publicly available, the quality is highly variable and BUSCO completeness poor with the exception of Procambarus virginalis (87.0 %) and Litopeneaus vannamei (84.6 %) (Table 1). It is now apparent that the assembly and annotation of decapod genomes are highly challenging due to the need to deal with large repetitive genomes, often with high heterozygosity, and should not to be undertaken by the faint-hearted or on a shoe-string budget (Zhang et al., 2019).The lack of genomic resources coupled with limited understanding of the molecular basis of gene expression and phenotypic variation will continue to limit advances in aquaculture-based productivity of decapods. Understanding the molecular basis of phenotypic variation and gene function is therefore important for selective breeding programs for traits such as increased growth and disease resistance (Zenger et al., 2019). Similarly, whole genome assemblies support genome-wide association studies (GWAS) to identify trait-specific loci and for genomic-based selective breeding.Freshwater crayfishes make up an important group of Decapod crustaceans (Crandall and De Grave, 2017) naturally occurring on all continents, with the exception of Antarctica and Africa (Toon et al., 2010; Bracken-Grissom et al., 2014). They reach their greatest species richness in Australia and north America. A number of species are subject to aquaculture and recreational, and indigenous and bait fisheries, and others are of conservation concern due to a range of activities threatening vulnerable freshwater environments (Piper, 2000; Richman et al., 2015; CABI, 2019). The genomic architecture and karyotype evolution of freshwater crayfish is also of interest as they have some of the largest chromosome numbers (2n=200) recorded for invertebrate species (Tan et al., 2004). In Australia, freshwater crayfish species from the genus Cherax Erichson, known as smooth yabbies, include the largest commercial crayfish species in the world (Austin, 1987; 1996; Austin and Knott, 1996). Of these, the economically-important and best known is the red claw crayfish, Cherax quadricarinatus von Martens, distributed widely across northern Australia (Austin, 1996; FAO, 2019). The species is capable of growing to greater than 200 grams making it the second largest commercial species of Cherax (Austin, 1987) behind the marron (C. cainii Austin) (Austin and Ryan, 2002). The popularity of the red claw as an aquaculture and ornamental species and its adaptability has resulted in widespread translocation, resulting in an extensive global distribution and it is acknowledged as a major invasive species of inland aquatic ecosystems in the tropics (Ahyong and Yeo, 2007; Larson and Olden, 2013; Saoud et al., 2013). Due to its large size and ease with which it can be maintained in captivity, C. quadricarinatus is also increasingly being used as a model organism to address fundamental questions relevant to molecular biology, physiology, functional genomics and cell biology (Fern &amp; #225;ndez et al., 2012; Pamuru et al., 2012; Ventura et al., 2019).The genetics of C. quadricarinatus has been well studied using PCR-based approaches and recently, next generation sequencing has been used for mitogenomic and transcriptomic studies (Baker et al., 2008; Gan et al., 2016; Tan et al., 2016). In this paper, we present the first genome assembly for a southern hemisphere crayfish using a hybrid assembly approach utilising long Nanopore reads and short Illumina reads, which has proven to be an efficient and effective approach for the assembly and annotation of species with large and repetitive genomes (Austin et al., 2017; Tan et al., 2018; Gan et al., 2019; S &amp; #225;nchez-Herrero et al., 2019), but has only been minimally explored to support decapod assemblies (Van Quyen et al., 2020).We then benchmark our assembly against seven other published decapod genome assemblies and present a preliminary phylogenomic analysis for decapod crustaceans. Lastly, we use these data to investigate the occurrence and diversification of nuclear mitochondrial DNA (NUMTs) in decapod genomes, which is directly relevant to debates on the veracity of the most common approach to the DNA barcoding of life for molecular-based taxonomic identification and biodiversity assessment based on the mitochondrial cytochrome oxidase 1 (cox1) gene region (Hebert et al., 2003; Cristescu, 2014; deWaard et al., 2018).2. Materials and Methods2.1 Data generationTissue samples from two male C. quadricarinatus individuals were collected from Northern Territory in Australia, DWN1 from Rapid Creek and M2R2 from Charles Darwin University Aquaculture Centre, both located in Darwin following institutionally endorsed ethical, biosecurity and safety guidelines. Genomic DNA was extracted using E.Z.N.A. &amp; #174; Tissue DNA Kit (Omega Bio-tek) from tail muscle tissue. One microgram of DWN1 was sent to Macrogen, Korea for PCR-free library preparation and sequencing on the Illumina HiSeq platform. Additional Illumina sequencing on the Novaseq6000 was also performed at Deakin Genomics Centre using a PCR-based library constructed with NEBNext Ultra Illumina library preparation kit. Using tissue samples from DWN1 and M2R2 individuals, for each Nanopore library, approximately 1 ug of gDNA as measured by Qubit Fluorometer (Invitrogen) was processed using the LSK108 library preparation kit followed by sequencing on the R9.4 MinION flowcell. Nanopore data was basecalled with Albacore versions compatible with kits and flowcells used (discontinued, was available on https://community.nanoporetech.com) and adapter-trimmed with the Porechop tool (Wick, 2017).2.2 De novo assembly and scaffolding of the crayfish genomeRaw reads were pre-processed prior to assembly. The fastp tool (–poly _g _min _len 1) (Chen et al., 2018) was used to trim polyG sequences from the 3’ end of reads generated on the Illumina NovaSeq platform. All short reads were trimmed with Trimmomatic (Bolger et al., 2014) to eliminate adapters (ILLUMINACLIP:2:30:10) and low quality sequences (AVGQUAL:20), and subsequently assembled with the Platanus assembler (Kajitani et al., 2014) using default parameters. The repeat content and large size of this genome necessitated scaffolding of the assembly with several data types; these included the use of short and long reads in order to span gaps of various sizes, in addition to using data from a previous C. quadricarinatus transcriptome project (Tan et al., 2016) to stitch together scaffolds containing coding genes. A first level of scaffolding was performed with Platanus, which scaffolds the assembled contigs and further closes gaps with insert size and sequence information from short paired-end libraries. Subsequently, a second level of scaffolding was accomplished with long Nanopore reads using LINKS (Warren et al., 2015), which was run for 20 iterations with multiple distance values (d) and step of sliding window (t), in addition to set values for k-mer (-k 19), minimum number of links (-l 5) and maximum link ratio between two best pairs (-a 0.3). Detailed parameters used in each iteration is available as Data Sheet 1. Resulting scaffolds were polished with pilon v1.22 (–fix all) (Walker et al., 2014) for five iterations. A third scaffolding step was performed using C. quadricarinatus RNA-seq reads previously generated (ERP004477) (Tan et al., 2016). HISAT2 was used to align RNA-seq reads to the assembly, information from which was used by BESST (Sahlin et al., 2014) for further scaffolding. All assemblies were evaluated for ‘completeness’ based on the detection of single copy conserved genes (arthropoda _odb9) with BUSCO (Sim &amp; #227;o et al., 2015).2.3 Genome size estimationJellyfish (Mar &amp; #231;ais and Kingsford, 2011) was used to count the occurrence of 19-, 21- and 25-mers in pre-processed short reads, generating a k-mer frequency histogram that was uploaded to the GenomeScope webserver (max kmer coverage disabled) (Vurture et al., 2017) for an estimation of haploid genome size, heterozygosity and repetitive content.2.4 Gene prediction and functional annotationA first iteration of the MAKER v2.31.10 annotation pipeline (Cantarel et al., 2008) was run to produce initial gene models based on transcript and protein hints (est2genome=1; protein2genome=1) from C. quadricarinatus transcriptome sequences (Tan et al., 2016), protein sequences from UniProt-SwissProt (Consortium, 2018) and an additional set of proteins from four other published decapod genomes: Eriocheir sinensis (Song et al., 2016), Litopenaeus vannamei (Zhang et al., 2019), Marsupenaeus japonicus and Penaeus monodon (Yuan et al., 2018). These gene models were used to train two ab initio gene predictors, SNAP (Korf, 2004) and AUGUSTUS (Stanke et al., 2006), which were provided in a second MAKER iteration. Gene models resulting from this iteration were again used to retrain SNAP and AUGUSTUS before a third and final iteration of MAKER. Genes with Annotation Edit Distance values (AED) ≤ 0.5 were retained (Eilbeck et al., 2009). A small AED value points to a lesser degree of variance between the predicted gene and evidences/hints used in prediction. For functional annotation, DIAMOND (Buchfink et al., 2014) was used to find homology between the predicted genes and known proteins in the UniProtKB (Swiss-Prot and TrEMBL) database (Consortium, 2018). The predicted protein sequences were also scanned for motifs, signatures and protein domains using InterProScan (Jones et al., 2014).2.5 Comparative analyses of published decapod genomesPublicly-available genome assemblies for seven published decapod species (Eriocheir sinensis, Exopalaemon carinicauda, Litopenaeus vannamei, Marsupenaeus japonicus, Neocaridina denticulata, Penaeus monodon, Procambarus virginalis) (Kenny et al., 2014; Song et al., 2016; Gutekunst et al., 2018; Yuan et al., 2018; Li et al., 2019; Zhang et al., 2019) and three other non-decapod crustaceans (Daphnia pulex, Eulimnadia texana, Parhyale hawaiensis) (Colbourne et al., 2011; Kao et al., 2016; Baldwin-Brown et al., 2017) were downloaded (sources in Table 1) and compared in terms of genome and assembly sizes, repeat content and completeness. ‘Completeness’ of a genome was assessed with BUSCO (Sim &amp; #227;o et al., 2015) based on the arthropoda _odb9 databases. To obtain repeat profiles, repeat families were identified de novo using RepeatModeler (Smit and Hubley, 2019), which uses RepeatScout (Price et al., 2005) and RECON (Bao and Eddy, 2002) to scan for repeats. The output from this step contains a set of consensus sequences from each identified repeat family that was used by RepeatMasker (Smit et al., 2019) to mask repeats within the assembly.2.6 Preliminary detection of NUMTsThe eight assembled genomes for C. quadricarinatus, Er. sinensis, Ex. carinicauda, L. vannamei, M. japonicus, N. denticulata, Pe. monodon and Pr. virginalis were also scanned for the presence of putative NUMTs. The blastn tool (Altschul et al., 1990) was used to align the mitochondrial cox1 gene of each species to its corresponding genome (exception: Procambarus clarkii cox1 was used since the full cox1 sequence for Pr. virginalis is unavailable on NCBI). Each output was filtered to eliminate alignments based on the following criteria:1. Alignment length &lt; 100 nucleotides2. Alignment length that spans 95 % of a genomic scaffold3. Alignment at the edge of a genomic scaffoldAn illustration of how these filters were applied is provided in Data Sheet 6. Sequences of these putative NUMTs were extracted based on alignment start and stop coordinates and were aligned with MAFFT (Katoh and Standley, 2013). IQ-TREE (Nguyen et al., 2014) was used to perform model testing and construct a maximum-likelihood (ML) phylogenetic tree (-m TESTMERGE).2.7 Phylogenetic analysesA ML tree was also constructed based on orthologous protein sequences predicted by BUSCO. This tree consists of six decapods (C. quadricarinatus, Er. sinensis, L. vannamei, M. japonicus, Pe. monodon, Pr. virginalis) with D. pulex, Eu. texana and Pa. hawaiensis as non-decapod outgroup species, rooted with Drosophila melanogaster. Since this analysis used sequences predicted by BUSCO, N. denticulata and Ex. carinicauda with low BUSCO completeness were excluded from this analysis. Briefly, protein sequences within each orthologous group (based on metazoa _odb9 and arthropoda _odb9) were aligned with MAFFT (Katoh and Standley, 2013) and trimmed with Gblocks (Castresana, 2000). Only multiple sequence alignments containing sequences from all ten species were retained and concatenated into a supermatrix with FASconCat-G (K &amp; #252;ck and Longo, 2014). IQ-TREE (Nguyen et al., 2014) was used for initial model testing and construction of the ML tree (-m TESTMERGE), with support values indicated by the SH-like approximate likelihood ratio test (SH-aLRT) (Guindon et al., 2010) and ultrafast bootstrap (UFBoot) (Minh et al., 2013) (-alrt 1000 -bb 1000).3. Results and Discussion3.1 A giant genome for a large freshwater crayfishThis study produced 191 Gbp of HiSeq data (2 &amp; #215;100 bp) and 774 Gbp of NovaSeq data (2 &amp; #215;150 bp) in addition to 36 Gbp of Nanopore reads (average: 3,419 bp) to assist with scaffolding. Kmer-based methods estimated a 5 Gbp size for the Cherax quadricarinatus genome (Data Sheet 2), a value that is within the 3.82 to 6.06 Gbp size range that has been reported for species in the Infraorder Astacidea (Gregory, 2019) (Figure 1A). Based on this estimate, data generated in this study yields sequencing depths of 193 &amp; #215; and 7 &amp; #215; of short and long reads, respectively. Sequence data is available in the Sequence Read Archive (SRA) on NCBI (BioProject: PRJNA559771).3.2 Characteristics of the C. quadricarinatus genomeHybrid assembly of this genome resulted in a 3.24 Gbp final assembly contained in 508,682 scaffolds, with a N50 length of 33,235 bp (Table 1). Details of each assembly at each scaffolding level are available in Data Sheet 3. The BUSCO tool reports the presence of 81.3 % and 12.8 % of complete and fragmented arthropod BUSCOs, respectively. These values are comparable to the completeness evaluation of other recently sequenced decapod genomes that are relatively much smaller (L. vannamei, Pr. virginalis, Er. sinensis) (Table 1, Figure 1B). GenomeScope profiles display a small shoulder on k-mer distributions, indicating some level of heterozygosity within the genome and estimated an average heterozygosity level of 0.44 %, which translates into approximately 1 mutation in 230 bp (Data Sheet 2). This is on par with the heterozygosity level of 0.53 % reported for the marbled crayfish, Pr. virginalis (Gutekunst et al., 2018), and is higher than in prawns with both M. japonicus and Pe. Monodon, both with 0.19 % and 0.21 % heterozygosity, respectively (Yuan et al., 2018). In addition, 33.73 % of this assembly was masked for repeats, with a large proportion of interspersed repeats being long interspersed nuclear elements (LINEs), representing a repeat profile that is similar to that of Pr. virginalis, another crayfish species (Figure 1C). While the report of only 5.9 % missing BUSCO genes in this C. quadricarinatus assembly suggests that majority of the gene space is present, the assembly reported in this study makes up only 64.8 % of the expected 5 Gbp genome with 85.2 % of short reads successfully mapped back to the assembly, indicating that there remains regions of this large genome that are missing from the assembly. This is not unusual for decapod genomes as they are rarely assembled to their full size, as seen from data points in Figure 1A, which mostly fall well below the “line of identity” between assembly vs genome size. The exception being Pr. virginalis with an assembled size close to its 3.5 Gbp genome size. This study tackles the ambitious assembly of a gigantic genome riddled with large repetitive structures and heterozygous regions that pose serious challenges to computational and bioinformatic resources, but there are clear benefits to hybrid assembly as an efficient method to add to the limited pool of genomic resources available for the order Decapoda. This assembly is available on NCBI (BioProject: PRJNA559771, WGS: VSFE00000000). It is noteworthy that this study has incorporated the largest dataset of long Nanopore reads to date for a decapod genome assembly (36 Gbp, textasciitilde7 &amp; #215;), the other being the recent updated assembly of the black tiger prawn genome that incorporated the use of 2.5 Gbp of long Nanopore reads ( textasciitilde1.25 &amp; #215;) (Van Quyen et al., 2020). The only other study to use long reads in decapod whole genome research is by Zhang et al. (2019) who utilised PacBio reads to support the assembly of the genome for the Pacific white shrimp L. vannamei.3.3 Predicted genes and functionalAn average of 95.4 % of RNA-seq reads from five C. quadricarinatus tissue types in Tan et al. (2016) were successfully aligned to the assembly. The annotation process predicted a total of 19,494 protein-coding genes (Data Sheet 3), with an average gene length of 9,768 bp containing an average of 4.4 exons per gene. This number of predicted genes is within the range of that typically reported for other decapod crustaceans (Er. sinensis: 14436, M. japonicus: 16716, Pe. monodon: 18100, Pr. virginalis: 21772, L. vannamei: 25596). Of the annotated protein-coding genes, 88 % are functionally annotated based on homology to existing UniProt protein sequences or through identification of protein domains and signatures. Protein and transcript sequences, BLAST homology alignments and InterProScan results are available as Data Sheet 4.3.4 Decapod evolution based on nuclear genesAlignment and trimming of 97 orthologous genes resulted in a supermatrix of 16,679 amino acid characters. Maximum-likelihood analysis based on these protein sequences generated the tree shown in Figure 1D. Rooted with the fruit fly, the phylogeny recovers branchiopods and amphipods as outgroup species to the decapod species. Shrimp and prawn species (L. vannamei, M. japonicus, Pe. monodon) are clustered in a clade as the Dendrobranchiata, which in turn forms a sister relationship with a second clade consisting of the other three decapod species. In this latter clade, crayfish species are placed as sister taxa (C. quadricarinatus and Pr. virginalis), consistent with their taxonomic status as species within Astacidea, but with surprisingly weak nodal support (SH-aLRT 55.1 %, UFBoot 66.0 %). These relationships are consistent with findings reported in mitogenome- and nuclear-based studies (Bracken et al., 2009; Shen et al., 2013; Tan et al., 2019; Wolfe et al., 2019). Alignment and phylogenetic tree produced in this analysis are available as Data Sheet 5.3.5 Integration of the mitochondrial cox1 gene into decapod genomesThe concept of DNA barcoding, by which a short universal DNA sequence is used to discriminate among species, is being very widely used to support taxonomic identification, detection of cryptic species and for establishing a reference database to support ecological, biodiversity and conservation-related studies (Ratnasingham and Hebert, 2007). A relatively recent and potentially powerful application of barcoding is the use of the rapidly increasing COI database as a reference for supporting environmental DNA metabarcoding studies (Deiner et al., 2017; G &amp; #252;nther et al., 2018). However, DNA barcoding has its critics (Moritz and Cicero, 2004; Collins and Cruickshank, 2013), with a persistent concern being the impact of mitochondrial DNA copies in the nuclear genome (Bensasson et al., 2001; Hazkani-Covo et al., 2010) on the veracity of the DNA barcoding methodology, potentially making DNA barcoding unreliable for certain taxonomic groups (Sorenson and Quinn, 1998), including crustaceans (Song et al., 2008).NUMTs have now been reported in a diversity of animal groups including crustaceans, with cytochrome oxidase 1 (cox1) NUMTs found in planktonic copepods (Bucklin et al., 1999) and snapping shrimp (Williams and Knowlton, 2001). Nguyen et al. (2002) and Munasinghe et al. (2003) reported the first NUMTs for crayfish, from the genus Cherax, and Song et al. (2008) for species of Orconectes. Our study extends the findings of Song et al. (2008) in that we identify cox1 pseudogenes in each of the decapod genome assemblies we have examined, but find the number to be widely variable among taxa. This analysis identified the most cox1 pseudogenes in C. quadricarinatus (34 insertion sites), followed by Pe. monodon (11), Pr. virginalis (8), M. japonicus (6), Ex. carinicauda (5), N. denticulata (3), Er. sinensis (2) and L. vannamei (1) (Figure 1E). NUMTs identified from these decapod species form monophyletic groups by species in the phylogeny, suggesting that the cox1 mitochondrial gene was integrated into each nuclear genome subsequent to the divergence of species in this analysis, with either multiple independent transfer events or further evolution of NUMTs within each species through duplication events. The NUMT phylogenetic tree is available in Data Sheet 5.In a broad-based study of NUMTs in 85 eukaryotic genomes, Hazkani-Covo et al. (2010) found a correlation with genome size. While finding the highest number of NUMTs in the red claw crayfish, with the second largest decapod genome assembled, is consistent with this observation, the evidence for a similar overall trend is not so clear across all the decapod species. In this context, it should be noted that we only have a relatively small sampling of decapod genomes and the varying assembly quality and volume of sequence data among studies makes testing this hypothesis fraught at this stage. There are also several caveats associated with this analysis. The identification process is dependent on the search methods and implemented filters, which if too conservative, can limit detection and exclude true NUMTs with highly divergent and variable sequences, especially if the time of insertion form the mitogenome occurred millions of years ago (Tsuji et al., 2012). It is also unknown whether other published decapod studies have been post-processed to exclude scaffolds containing mitochondrial genes during the submission process to databases. Nevertheless, results from this analysis provide an insight into the prevalence of NUMTs in decapod genomes, encouraging further exploration into the evolution of NUMTs in future studies.4. ConclusionFollowing on from mitogenomic and transcriptomic studies, we present the first draft genome for the red claw crayfish (Cherax quadricarinatus) based on relatively large volumes of short and long genomic reads from Illumina and Oxford Nanopore (ONT) platforms. While the assembly is relatively fragmented, it is much better than many of those currently available for decapod species, and the quality of the annotation is equivalent to other more recently-sequenced decapod genomes. Due to the very large size and repetitive structures of the red claw genome, the assembly was highly challenging. However, we demonstrated the value of long Nanopore reads and a hybrid assembly approach for improving an assembly based on short reads alone (Data Sheet 3), which gives encouragement for tackling other crayfish and crustacean taxa with large and complex genomes. This draft genome will be an important and valuable resource to support ongoing comparative genomic, phylogenomics and molecular-based breeding studies for aquaculture, conservation and biodiversity-related studies and can be approved upon over time, with the generation of additional long read data. However, a major challenge still remains in relation to the computational resources needed to assemble large repetitive genomes from predominately short reads, even when aided with long reads (Lewin et al., 2018). Computationally, assembly, scaffolding and polishing processes to achieve the final draft genome took almost 85 processor-weeks, and annotation required another 100 processor-weeks. A worthwhile next step would be to investigate the efficacy of a long-read led crayfish genome assembly, now feasible as result of declining costs and improved accuracy of long read sequencing, and which should be less expensive and lead to greatly reduced processor time for assembly tasks.Data availabilityThe datasets generated for this study can be found in the NCBI BioProject PRJNA559771, BioSamples SAMN12558911 and SAMN12885547 (WGS accession: VSFE00000000; SRA accessions: SRR10214688, SRR10416208, SRR10416210, SRR10445782, SRR10445783, SRR10445784, SRR10484712).Author ContributionsCA, HG and LC conceived the project. CA collected the samples, HG and YL performed the sequencing and MT analysed the data. MT and CA wrote the manuscript. FG and LC modified the manuscript. All authors read and approved the final version of the manuscript.FundingFunding for this study was provided by the Deakin Genomics Centre, Deakin University (Australia), by the Centre National de la Recherche Scientifique and the University of Poitiers (France) and the Monash University Malaysia Tropical Medicine and Biology Platform, Monash University (Malaysia).</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tan_giant_2020</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A {Giant} {Genome} for a {Giant} {Crayfish} ({Cherax} quadricarinatus) {With} {Insights} {Into} cox1 {Pseudogenes} in {Decapod} {Genomes}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-8021}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2020.00201/full}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fgene.2020.00201}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2025-10-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Genetics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tan, Mun Hua and Gan, Han Ming and Lee, Yin Peng and Grandjean, Frederic and Croft, Laurence J. and Austin, Christopher M.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Aquaculture, Freshwater crayfish, Genome, hybrid assembly, Illumina, Oxford nanopore, Parastacidea}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="baudry_when_2024" class="col-sm-8"> <div class="title">When Methodological Innovation Changes the Game: A 10-Year Review of Environmental DNA (eDNA) Applied to Crayfish</div> <div class="author"> Thomas Baudry, Valentin Vasselon, Carine Delaunay, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Michael Sweet, Frédéric Grandjean' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Aquatic Conservation: Marine and Freshwater Ecosystems</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/aqc.4245" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>The use of environmental DNA (eDNA) as a tool for monitoring represents a major innovative advance in environmental science, one that enables the detection of species without the need to observe or capture them. This article assesses the state of play of eDNA research targeting crayfish. We found a total of 41 peer-reviewed articles published between 2014 and 2023 on both native and invasive species. Most studies focused on invasive species (or a native/invasive species co-detection assessment) (65.8 %). There was also a clear geographical bias across studies, with more than half conducted in Europe (51.2 %) and a quarter in the United States (26.8 %). In contrast, there were none conducted in Africa. The relatively large number of published studies has led to an interesting diversity of protocols designed or utilized, with most favouring the development of their own assays (69.33 %). That said, filtration (as an eDNA capture method) was common (80.5 %), along with the use of commercially available eDNA extraction kits (69.8 %). The COI gene also appeared to be the preferred target region (89.33 %). Such range of protocols is interesting, but is it optimal? Are the best protocols always being utilized? Or is the chance for novel application hampering our ability to explore larger trends across studies?</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">baudry_when_2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{When {Methodological} {Innovation} {Changes} the {Game}: {A} 10-{Year} {Review} of {Environmental} {DNA} ({eDNA}) {Applied} to {Crayfish}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{34}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2024 The Author(s). Aquatic Conservation: Marine and Freshwater Ecosystems published by John Wiley &amp; Sons Ltd.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1099-0755}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{When {Methodological} {Innovation} {Changes} the {Game}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/aqc.4245}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/aqc.4245}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2025-10-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Aquatic Conservation: Marine and Freshwater Ecosystems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Baudry, Thomas and Vasselon, Valentin and Delaunay, Carine and Sweet, Michael and Grandjean, Frédéric}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{freshwater conservation, geographical biases, invasive/native species, PCR methods, protocol variety}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e4245}</span><span class="p">,</span>
  <span class="na">annote</span> <span class="p">=</span> <span class="s">{e4245 AQC-24-0057.R2}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%66%72%65%64%65%72%69%63.%67%72%61%6E%64%6A%65%61%6E@%75%6E%69%76-%70%6F%69%74%69%65%72%73.%66%72" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://cv.hal.science/frederic-grandjean/" title="HAL" rel="external nofollow noopener" target="_blank"><i class="ai ai-hal"></i></a> <a href="https://orcid.org/0000-0002-8494-0985" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://www.researchgate.net/profile/Frederic-Grandjean-3/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="https://scholar.google.com/citations?user=jq6-1DMAAAAJ&amp;hl" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.scopus.com/authid/detail.uri?authorId=7005355159" title="Scopus" rel="external nofollow noopener" target="_blank"><i class="ai ai-scopus"></i></a> </div> <div class="contact-note"></div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Frédéric Grandjean. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/TEST/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/TEST/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/TEST/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/TEST/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/TEST/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/TEST/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/TEST/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/TEST/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/TEST/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/TEST/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/TEST/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/TEST/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/TEST/assets/js/search-data.js"></script> <script src="/TEST/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>